src/components/app-list-item/style.css

```diff
@@ -69,15 +68,58 @@
   content: url('!~@fortawesome/fontawesome-free/svgs/solid/cog.svg');
   display: block;
   height: 4rem;
-  margin-top: calc(-2rem + var(--space-3)  / 2); /* 1 */
+  margin-top: calc(-2rem + var(--space-3) / 2); /* 1 */
   opacity: 0.1;
   position: absolute;
   top: 50%;
   right: -2rem;
   width: 4rem;
 }
 
+@keyframes animate-placeholder {
```

Comment:
If the animation is used only in this component (which it is) then its name should also be namespaced, i.e. `@keyframes app-list-item-animate-placeholder`